docs/bcfusion/validators/key-managment.md

```diff
@@ -0,0 +1,53 @@
+# Key Management for BSC Validators
+
+[BEP-294](https://github.com/bnb-chain/BEPs/pull/294) and [BEP-297](https://github.com/bnb-chain/BEPs/pull/297)
+introduce the native staking and governance features for BNB Smart Chain (BSC). For a validator, when participating in
+staking (e.g., creating a validator, self-delegating) and governance, there are several wallet keys that will be
+involved. To help validators manage their keys and funds effectively and safely, the following practices are
+recommended.
+
+## Operator Key
+
+The operator key is used for operating a validator, including creating a validator, editing the information of a
+validator, and undelegating. When creating a validator, the operator key is also used for self-delegating with more than
+2000 BNB. When interacting with the new BSC staking dApp, the operator key is mostly involved.
```

Comment:
not 2000BNB, but 2001 BNB, one is lock to dead address, the extra 1 BNB will be burned.